accessibilite/l-equipe-a-une-personne-avec-la-competence-front-accessibilite.md

```diff
@@ -1 +1,14 @@
-# L'équipe a une personne avec la compétence front accessibilité.
\ No newline at end of file
+# L'équipe a une personne avec la compétence front accessibilité.
```

Comment:
@astranchet on trouvait que c'était un peu engagé comme règle avec @revolunet , qu'en pense-tu ?

Reply:
moi ça me parait cohérent ici c'est plus sur le standard UX que j'ai des doutes

Reply:
Moi j'adouçis avec ça OU utiliser le DSFR. Mais je sais pas comment le formuler en markdown. Peut être reformuler en mode "Le code front est sémantique" ou qqchose comme ça ?